sdk/src/org.graalvm.nativeimage/src/org/graalvm/nativeimage/Platform.java

```diff
@@ -336,6 +336,22 @@ public IOS_AARCH64() {
         }
     }
 
+    /**
+     * Supported leaf platform: iOS on x86 64-bit.
+     *
+     * @since 21.1
```

Comment:
Also since 21.1 has already been released, which version should I use here?

Reply:
21.3